skt/executable.py

```diff
@@ -534,6 +535,11 @@ def setup_parser():
         default=False,
         help="Build kernel with debuginfo (default: disabled)"
     )
+    parser_build.add_argument(
+        "--kbuild-output",
```

Comment:
The argument name is fine, I just wonder if we could shorten it to '-o' or is that to confusing?

Reply:
I was just trying to be specific and verbose there. :)